flang-rt/lib/runtime/reduction.cpp

```diff
@@ -374,14 +380,14 @@ void RTDEF(AnyDim)(Descriptor &result, const Descriptor &x, int dim,
       result, x, dim, terminator, "ANY");
 }
 
-std::int64_t RTDEF(Count)(
+RT_API_ATTRS std::int64_t RTDEF(Count)(
```

Comment:
nit: `RT_API_ATTRS` is embedded into `RTDEF` and `RTDECL`, so it is redundant here and below and in `reduction.h`

Reply:
I didn't notice that. Thanks. I just removed the redundant part.